utils/aggregator/variables.tf

```diff
@@ -0,0 +1,45 @@
+variable "conf_list" {
+  description = "List of module output with the config"
```

Comment:
Maybe add the precedence rules in the description

Reply:
Don't get the point. Are you talking about the precedence order  in general or is it something specific to this variable?

Reply:
I mean explain if the first element in the list has precedence or the last

Reply:
there is no precedence rule. This brings the question if there is a duplicate information in different elements, even if that is most not likely to happen

Reply:
there is precedence dictated by the terraform merge function